src/pages/CategoriesPage/DeleteCategory/DeleteCategoryComponent.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  Button,
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  Grid,
+} from '@mui/material';
+import { styles } from '../styles';
+
+interface Props {
+  deleteModalOpen: boolean;
+  onToggleDeleteStatusClick: () => void;
+  onSubmit: () => void;
+}
+
+const DeleteCategoryComponent = (props: Props) => {
+  return (
+    <Dialog open={props.deleteModalOpen}>
+      <div style = {styles.dialogTitleWrapper}>
+        <DialogTitle style={styles.dialogTitle}>
+          Click on the confirmation button to delete the category
+        </DialogTitle>
+      </div>
+      <DialogContent>
```

Comment:
It is not content. You should use  <DialogActions> for buttons https://mui.com/material-ui/react-dialog/#ResponsiveDialog.tsx

Reply:
New location for buttons is <DialogActions> component